### DISCUSSION TITLE
Assimp::Importer read blend file filed

### DISCUSSION THREAD

**Discussion by RiverUp2012** [assimp/assimp#5912] [General]
Assimp::Importer read blend file filed, error:Failure resolving pointer 0x604971f1, no file block falls into this address range.why?

![image](https://github.com/user-attachments/assets/a19c3911-addc-4bb1-b7ed-4245420fb69b)

[BlenderDefault_262.zip](https://github.com/user-attachments/files/18033601/BlenderDefault_262.zip)

**Comment by tellypresence** [comment]
Confirm error on attempt to load attached model file
```
Failure resolving pointer 0x1615426033, no file block falls into this address range
```
Suspect that model file is corrupt or too new for assimp's blender loader implementation
